src/pages/courseDetailsPage/CourseDetailsPage.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from "react";
+import { useLocation, useParams } from "react-router-dom";
+import { useGetCourseByIdQuery } from "redux/coursesApi";
+import {
+  ArrowIcon,
+  BackLink,
+  CourseDescription,
+  CourseLesson,
+  CourseLessonsList,
+  LessonTitle,
+  CourseTitle,
+  LockIcon,
+  Title,
+  TitleBox,
+  VideoContainer,
+  CurrentLesson,
+  CurrentLessContainer,
+  CurrentText,
+} from "./CourseDetailsPage.styled";
+import { Container } from "components/coursesList/CoursesList.styled";
+import VideoPlayer from "components/VideoPlayer/VideoPlayer";
+
+interface LessonseDetails {
+  id: string;
+  link: string;
+  title: string;
+  status: string;
+  previewImageLink: string;
+  order: number;
+}
+
+const CourseDetailsPage: React.FC = () => {
+  const { id: courseId = "" } = useParams();
```

Comment:
|my opinion|
Use single quotes
[link to article](https://flexiple.com/javascript/double-vs-single-quotes-javascript/#:~:text=a%20single%20quote.-,Single%20vs%20Double%20Quotes%20%2D%20Pros%20and%20Cons,-Pros)

Reply:
I use double quotes